resources/views/themes/default1/front/auth/login-register.blade.php

```diff
@@ -1497,9 +1501,12 @@ function getState(val) {
 
         function getCode(val) {
             $.ajax({
+                headers: {
+                   'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+                },
                 type: "GET",
                 url: "{{url('get-code')}}",
-                data: {'country_id':val,'_token':"{{csrf_token()}}"},//'country_id=' + val,
+                data: {'country_id':val},//'country_id=' + val,
```

Comment:
@sowmiyasubramaniyan you have to csrf token to the form. Not just remove it from the URL. Now this is vulnerable to cross-site attack

Reply:
![image](https://user-images.githubusercontent.com/96425427/168213618-8fee0175-1821-473c-a633-a4146db626be.png)